R/runEmployment.R

```diff
@@ -0,0 +1,304 @@
+#' Computes the employment values (jobs) across different sectors
+#' @param pathToMIF A mif file putput from a REMIND run
+#' @param improvements Either "None", "CEEW", "Dias", "Rutovitz_aus","Solar_found" or "All". Use "All" for all improvements.
+#' @param subtype Subtype for how shares of solar rooftop, wind offshore, and small hydro are assumed in the future. Options "current", "irena", and "expert". See calcDspvShare for more information.
+#' @param share_manf Either "current" or "local". Current implies current shares of world manufacture remain same until 2050, current means that in 2050 all countries manufacture required components locally.
+#' @param multiplier controls how the regional multiplier for non-oecd countries changes with time.
+#' @param decline How should the employment factors change over time? "capcosts" means according to capital costs. "static" means it doesn't change
+#' @description This function returns a magpie object containing the reporting the jobs for different technologies
+#' @return A magpie object
+#' @author Aman Malik
+#' @examples
+#' \dontrun{
+
+#' runEmployment(pathToMIF, improvements = "All", multiplier = "own", subtype = "expert", share_manf = "local", decline = "capcosts")
```

Comment:
Examples must not have lines with more than 100 characters, or they will get a NOTE which prevents buildLibrary from successfully finishing.
```suggestion
#' runEmployment(pathToMIF, improvements = "All", multiplier = "own", subtype = "expert",
#'               share_manf = "local", decline = "capcosts")
```

Reply:
100 or 120? lucode tells me 120 characters.

Reply:
In code it's 120 which is how we configured our linter. For examples the standard R CMD CHECK defines the limit as 100.